src/CommunityToolkit.Maui.Analyzers.UnitTests/Verifiers/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -9,11 +9,11 @@ public static partial class CSharpCodeFixVerifier<TAnalyzer, TCodeFix>
 	where TAnalyzer : DiagnosticAnalyzer, new()
 	where TCodeFix : CodeFixProvider, new()
 {
-	protected class Test : CSharpCodeFixTest<TAnalyzer, TCodeFix, DefaultVerifier>
+	class Test : CSharpCodeFixTest<TAnalyzer, TCodeFix, DefaultVerifier>
 	{
 		public Test(params ReadOnlySpan<Type> assembliesUnderTest)
 		{
-#if NET9_0
+#if NET10_0
 			ReferenceAssemblies = Microsoft.CodeAnalysis.Testing.ReferenceAssemblies.Net.Net90;
```

Comment:
Reference assemblies net10?

Reply:
Microsoft hasn't yet released `Microsoft.CodeAnalysis.Testing.ReferenceAssemblys.Net.Net100`.

I've updated our build script to skip testing `CommunityToolkit.Maui.Analyzers.UnitTests` for now until Microsoft releases it.